### PR TITLE
Update template to cumulus@6ed9632

### LINKED ISSUE #19: no matching package named `cumulus-pallet-xcm-handler` found
no matching package named `cumulus-pallet-xcm-handler` found
```
    Updating crates.io index
    Updating git repository `https://github.com/paritytech/cumulus.git`
    Updating git repository `https://github.com/paritytech/substrate.git`
    Updating git repository `https://github.com/paritytech/polkadot`
error: no matching package named `cumulus-pallet-xcm-handler` found
location searched: https://github.com/paritytech/cumulus.git?branch=rococo-v1
```

### PR DESCRIPTION
**Pull request by stanly-johnson**
Compile to `cumulus@6ed9632`
Parachain builds blocks with `polkadot@943038a`

```
2021-04-15 11:43:30.880 TRACE tokio-runtime-worker runtime: [Relaychain] Batch-verification: 0 pending tasks, 0 sr25519 signatures    
2021-04-15 11:43:30.880 TRACE tokio-runtime-worker runtime: [Relaychain] Finalization of batch verification took 0 ms    
2021-04-15 11:43:30.921  INFO tokio-runtime-worker substrate: [Relaychain] ✨ Imported #80 (0x5a79…3e29)    
2021-04-15 11:43:34.585  INFO tokio-runtime-worker substrate: [Relaychain] 💤 Idle (3 peers), best: #80 (0x5a79…3e29), finalized #77 (0x2ffe…ca53), ⬇ 0.5kiB/s ⬆ 0.2kiB/s    
2021-04-15 11:43:34.675  INFO tokio-runtime-worker cumulus-collator: [Parachain] Starting collation. relay_parent=0x5a79416db09d4f464694d17f455770244363e3f45aa6c48f3d92fba38fb43e29 at=0x945e31224a573846aa6266513c4c0c8d9de38443c9a6ff0e8c7053c6caaf2e9b
2021-04-15 11:43:34.675  INFO tokio-runtime-worker sc_basic_authorship::basic_authorship: [Parachain] 🙌 Starting consensus session on top of parent 0x945e31224a573846aa6266513c4c0c8d9de38443c9a6ff0e8c7053c6caaf2e9b    
2021-04-15 11:43:35.130  INFO tokio-runtime-worker sc_basic_authorship::basic_authorship: [Parachain] 🎁 Prepared block for proposing at 5 [hash: 0xed37da1339b639496908c9a5fceb5b9a6f0540c89d1b45eaf3ce97a16dcf5bc7; parent_hash: 0x945e…2e9b; extrinsics (2): [0xfc99…940a, 0x74bc…0ddb]]    
2021-04-15 11:43:35.132  INFO tokio-runtime-worker substrate: [Parachain] ✨ Imported #5 (0xed37…5bc7)    
2021-04-15 11:43:35.133  INFO tokio-runtime-worker cumulus-collator: [Parachain] Produced proof-of-validity candidate. block_hash=0xed37da1339b639496908c9a5fceb5b9a6f0540c89d1b45eaf3ce97a16dcf5bc7
2021-04-15 11:43:35.341  INFO tokio-runtime-worker substrate: [Parachain] 💤 Idle (0 peers), best: #4 (0x945e…2e9b), finalized #2 (0x03d5…a831), ⬇ 0 ⬆ 0    
2021-04-15 11:43:36.164  INFO tokio-runtime-worker babe: [Relaychain] 👶 New epoch 8 launching at block 0xfb21…ebae (block slot 269747836 >= start slot 269747836).    
2021-04-15 11:43:36.164  INFO tokio-runtime-worker babe: [Relaychain] 👶 Next epoch starts at slot 269747846    
2021-04-15 11:43:37.246 TRACE tokio-runtime-worker runtime: [Relaychain] Batch-verification: 0 pending tasks, 0 sr25519 signatures    
2021-04-15 11:43:37.247 TRACE tokio-runtime-worker runtime: [Relaychain] Finalization of batch verification took 0 ms    
2021-04-15 11:43:37.369  INFO tokio-runtime-worker substrate: [Relaychain] ✨ Imported #81 (0xfb21…ebae)    
2021-04-15 11:43:39.585  INFO tokio-runtime-worker substrate: [Relaychain] 💤 Idle (3 peers), best: #81 (0xfb21…ebae), finalized #78 (0xb701…1c11), ⬇ 1.6kiB/s ⬆ 2.4kiB/s    
2021-04-15 11:43:40.341  INFO tokio-runtime-worker substrate: [Parachain] 💤 Idle (0 peers), best: #4 (0x945e…2e9b), finalized #3 (0x9239…2e9a), ⬇ 0 ⬆ 0    
2021-04-15 11:43:41.112  INFO tokio-runtime-worker cumulus-collator: [Parachain] Starting collation. relay_parent=0xfb2130a12097367adecd282b24d696050575f37a7d1b1e77bc8080b98682ebae at=0x945e31224a573846aa6266513c4c0c8d9de38443c9a6ff0e8c7053c6caaf2e9b
2021-04-15 11:43:41.112  INFO tokio-runtime-worker sc_basic_authorship::basic_authorship: [Parachain] 🙌 Starting consensus session on top of parent 0x945e31224a573846aa6266513c4c0c8d9de38443c9a6ff0e8c7053c6caaf2e9b    
2021-04-15 11:43:41.564  INFO tokio-runtime-worker sc_basic_authorship::basic_authorship: [Parachain] 🎁 Prepared block for proposing at 5 [hash: 0x8c984cd0d0427ecd486b7a1ed4606fac82d98d764dfee7b3352ec846c4fbde34; parent_hash: 0x945e…2e9b; extrinsics (2): [0xd8e2…4ffd, 0xd269…4149]]    
2021-04-15 11:43:41.566  INFO tokio-runtime-worker substrate: [Parachain] ✨ Imported #5 (0x8c98…de34)    
2021-04-15 11:43:41.566  INFO tokio-runtime-worker cumulus-collator: [Parachain] Produced proof-of-validity candidate. block_hash=0x8c984cd0d0427ecd486b7a1ed4606fac82d98d764dfee7b3352ec846c4fbde34
2021-04-15 11:43:42.911 TRACE tokio-runtime-worker runtime: [Relaychain] Batch-verification: 0 pending tasks, 0 sr25519 signatures    
2021-04-15 11:43:42.912 TRACE tokio-runtime-worker runtime: [Relaychain] Finalization of batch verification took 0 ms    
2021-04-15 11:43:42.961  INFO tokio-runtime-worker substrate: [Relaychain] ✨ Imported #82 (0xe3d1…83c7)    
```

Closes #19

### CODE REVIEW

File: node/src/command.rs [hunk 1]
```diff
@@ -199,7 +216,7 @@ pub fn run() -> Result<()> {
 
 			let block: Block = generate_genesis_block(&load_spec(
 				&params.chain.clone().unwrap_or_default(),
-				params.parachain_id.into(),
+				params.parachain_id.unwrap_or(100).into(),
```
Comment (riusricardo) [inline node/src/command.rs:219]:
```suggestion
				params.parachain_id.unwrap_or(200).into(),
```

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -360,10 +435,12 @@ construct_runtime!(
...
-		ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Storage, Inherent, Event},
+		ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Storage, Inherent, Event<T>},
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
 		ParachainInfo: parachain_info::{Pallet, Storage, Config},
-		XcmHandler: cumulus_pallet_xcm_handler::{Pallet, Event<T>, Origin},
+		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>},
+		PolkadotXcm: pallet_xcm::{Pallet, Call, Event<T>, Origin},
+		CumulusXcm: cumulus_pallet_xcm::{Pallet, Origin},
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>},
         TemplatePallet: template::{Pallet, Call, Storage, Event<T>},
```
Comment (riusricardo) [inline runtime/src/lib.rs:445]:
```suggestion
		TemplatePallet: template::{Pallet, Call, Storage, Event<T>},
```

**Review by riusricardo — COMMENTED** [review]
LGTM. Thanks @stanly-johnson 
@al3mart please take a look and validate the changes.

**Review by al3mart — APPROVED** [review]
Neat!

**Pull request merged**